pubsub/kafka/kafka.go

```diff
@@ -0,0 +1,169 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package kafka
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+
+	"github.com/Shopify/sarama"
+	"github.com/dapr/components-contrib/pubsub"
+	log "github.com/sirupsen/logrus"
```

Comment:
there is an change for log, please use "github.com/dapr/dapr/pkg/logger" instead of logrus.